src/config.c

```diff
@@ -629,6 +629,22 @@ void cgroup_config_cleanup_namespace_table(void)
 	memset(&config_namespace_table, 0, sizeof(struct cg_mount_table_s) * CG_CONTROLLER_MAX);
 }
 
+static int is_valid_controller(char *ctrl)
+{
+	int i;
+
+	static const char *ctrl_list[] = { "blkio", "cpu", "cpuacct", "cpuset", "devices",
+					"freezer", "hugetlb", "memory", "misc", "net_cls",
+					"net_prio", "perf_event", "pids", "rdma", NULL };
```

Comment:
This is a note to @kamalesh-babulal and myself.  We'll need to make sure that this list of valid controllers closely follows the upstream kernel.  If/When new controllers are added, we'll need to update this function.

Reply:
@drakenclimber Agreed, we need to keep an eye on new controllers.